salesforce_api/kafka_salesforce_integration/kafka_msgProducer/main.bal

```diff
@@ -0,0 +1,23 @@
+import ballerina/http;
+import ballerinax/kafka;
+
+public type ProductPrice readonly & record {|
+    string name;
+    float unitPrice;
+|};
+
+service / on new http:Listener(9090) {
+    private final kafka:Producer priceProducer;
+
+    function init() returns error? {
+        self.priceProducer = check new (kafka:DEFAULT_URL);
+    }
+
+    resource function post orders(@http:Payload anydata productPrice) returns http:Accepted|error {
+        check self.priceProducer->send({
+            topic: "foobar",
```

Comment:
instead of `foobar` try to have meaningful name based on sample context

Reply:
Updated in [bcf09ac](https://github.com/ballerina-guides/integration-samples/pull/31/commits/bcf09ac42e87543485854586312facacacd050cb) to `product_price_update`